src/influx.jl

```diff
@@ -0,0 +1,50 @@
+import JSON3
+import HTTP
+import DataFrames as DF
+import Dates: DateTime
+
+# NOTE: this doesn't actually do anything smart like batching
+# or keeping an open connection, it just remembers the connection
+# details
+struct InfluxClient
+    host::String
+    database::String
+    port::Int
+    path::String
+    username::String
+    password::String
+    InfluxClient(
+        host::String,
+        database::String,
+        port::Int = 8086,
+        path::String = "query",
+        username::String = "",
+        password::String = "",
+    ) = new(host, database, port, username, password, path)
+end
```

Comment:
I don't think this kind of convenience constructors should be inner.  You should move this to outer, then you cannot use `new`.  So, probably this:

```suggestion
end

InfluxClient(
    host::String,
    database::String,
) = InfluxClient(host, database, 8086, "query", "", "")
```